{"name": "test_login_params[data_for_login2]", "status": "passed", "attachments": [{"name": "log", "source": "82939a72-43a4-4ee0-90b3-fe91434e5cb6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh@gmail.com', '48465', 'Fail')"}], "start": 1745752033151, "stop": 1745752036645, "uuid": "2cc18161-8ca0-45c3-a99b-4b1ef1f39a68", "historyId": "cd1ac65cf3cc05caccdeb520d07aec8a", "testCaseId": "47f970e602e4eefb1a3e5455dd555a36", "fullName": "testCases.test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "17944-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login_params"}]}